Improve readability

malte-laukoetter opened the pull request:
I have formated the text to use JS template strings instead of "normal" strings, as these allow line breaks and direct variable replacement (with `${js expression}`) so it is a lot simpler to read or edit the text. Additionally I moved the variables to use `const` instead of `var` to first remove the multiple declarations of these and make clear that these shouldn't been modified (two common pitfall for bugs).

The resulting document should look exactly the same as beforehand.

malte-laukoetter committed:
Improve readability of the text

malte-laukoetter committed:
Improve readability

malte-laukoetter committed:
Fix text formating

karlbeecken commented on js/script.js:
```diff
@@ -1,66 +1,37 @@
+  const city = document.getElementById('city').value
+  const gender = document.getElementById('gender').value
 
+  const salutation = gender === 'woman' ? 'Sehr geehrte Frau' : 'Sehr geehrter Herr'
```
In the HTML file stands 'women' instead of 'woman', so you have to change it here OR in the HTML, whatever you like more. When you change the HTML, please also replace 'men' with 'man' for convenience ;) 
```suggestion
  const salutation = gender === 'women' ? 'Sehr geehrte Frau' : 'Sehr geehrter Herr'
```

karlbeecken reviewed (changes requested):
Much nicer coding style, but one mistake was made

malte-laukoetter committed:
Merge branch 'master' of github.com:fridaysforfuture/entschuldigungs-generator

malte-laukoetter committed:
Fix gender names

I decided to use male and female as these are the actual words for the
gender :)

malte-laukoetter commented:
Thanks for spotting the mistake, I fixed it and solved the merge conflict :)

karlbeecken commented:
I think female/male is a much better choice :+1:

karlbeecken commented:
@Lergin Line 95 was a very smart idea, props on this!
